### DISCUSSION TITLE
信息论与编码 | YuZhangWang的领域

### DISCUSSION THREAD

**Discussion by giscus** [YuZhangWang/YuZhangWang.github.io#15] [Announcements]
# 信息论与编码 | YuZhangWang的领域

这篇博客是帮忙测试的内容，博客主人关于公式渲染出了一些问题，我放上来进行测试使用  前言这篇总结和系统性学习地笔记，是基于电子科技大学的《信息论与编码》课程，而我写作的初衷是个人的复习。因此，内容可能不是那么详细，但是我会保证基本上我会在自己理解的基础上，完成这篇文章。随着我自己的理解的加深，可能会出现前后不一致的情况，如果读者感到矛盾，那么请在评论区说明，等我有时间时会继续完善的。 离散信源

https://yuzhang.wang/124-information-theory/

**Comment by BachWV** [comment]
很好，学习了
